obelisk/cpp/obelisk_cpp/CMakeLists.txt

```diff
@@ -0,0 +1,62 @@
+message(STATUS "Building Obelisk Cpp")
+
+if(CMAKE_COMPILER_IS_GNUCXX OR CMAKE_CXX_COMPILER_ID MATCHES "Clang")
+  add_compile_options(-Wall -Wextra -Wpedantic)
+endif()
+
+# ROS 2 Packages
+find_package(ament_cmake REQUIRED)
+find_package(rclcpp REQUIRED)
+find_package(rclcpp_lifecycle REQUIRED)
+find_package(rcl REQUIRED)
+
+# Obelisk Messages
+find_package(obelisk_control_msgs REQUIRED)
+find_package(obelisk_sensor_msgs REQUIRED)
+find_package(obelisk_estimator_msgs REQUIRED)
+
+# ------- Mujoco ------- #
+# Setup Mujoco path
+set(MUJOCO_VERSION "mujoco-3.1.6" CACHE STRING "mujoco version")
+set(MUJOCO_DIR "/home/$ENV{USER}/mujoco/${MUJOCO_VERSION}" CACHE PATH "Path to Mujoco dir")
```

Comment:
yup, this is tied to the above comments (and thus #34) - I'll do some more thinking about it today. At the end of the day though, there are a number of workable options so this will be somewhat of a preference thing.